snapshot/pkg/controller/snapshotter/snapshotter.go

```diff
@@ -38,15 +38,18 @@ import (
 )
 
 const (
+	snapshotMetadataTimeStamp        = "SnapshotMetadata-Timestamp"
+	snapshotMetadataPVName           = "SnapshotMetadata-PVName"
+	snapshotDataNamePrefix           = "k8s-volume-snapshot"
 	defaultExponentialBackOffOnError = true
 
 	// volumeSnapshot* is configuration of exponential backoff for
-	// waiting for snapshot operation to complete. Starting with 10
-	// seconds, multiplying by 1.2 with each step and taking 15 steps at maximum.
-	// It will time out after 12.00 minutes.
-	volumeSnapshotInitialDelay = 10 * time.Second
-	volumeSnapshotFactor       = 1.2
-	volumeSnapshotSteps        = 15
+	// waiting for snapshot operation to complete. Starting with 2
+	// seconds, multiplying by 2 with each step and taking 15 steps at maximum.
+	// It will time out after 15 steps.
+	volumeSnapshotInitialDelay = 2 * time.Second
+	volumeSnapshotFactor       = 1.5
+	volumeSnapshotSteps        = 20
```

Comment:
Based on my calculation, starting with 2 seconds (at step 0), and multiplying by 1.5 with each step, it will be 6650 seconds after 20 steps.  Do we really want to wait that long?

Reply:
I don't have data about how long a snapshot might take? But from what I heard is that is it possible snapshot might take hours..

Reply:
That's possible depending on how much data needs to be copied over.  So I'm fine with the total time here.